src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfig.java

```diff
@@ -164,6 +173,21 @@ public String getDedupeBy() {
     return Strings.isNullOrEmpty(dedupeBy) ? null : dedupeBy;
   }
 
+  @Nullable
+  public String getPartitionFilter() {
+    if (partitionFilter != null) {
+      partitionFilter = partitionFilter.trim();
+      if (partitionFilter.isEmpty()) {
+        return null;
+      }
+      // remove the WHERE from the filter
+      if (partitionFilter.toUpperCase().startsWith(WHERE)) {
```

Comment:
why is this needed?

Reply:
@CuriousVini, this is needed in case someone copy/paste or accidentally puts the WHERE keyword at the beginning of the Filter Partition textbox.